recipes/meg/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "meg" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lamyj/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 452b103f43544df9778b00491b9de3324807bcdeaace1bb24554ce39aed71ed2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy x.x
```

Comment:
`numpy` isn't needed to create the package.

Reply:
Requirement removed in 90cae7f0bf